_assets/javascripts/config.js

```diff
@@ -111,5 +112,10 @@ module.exports = [
   {
     name: "skeleton",
     files: ["components/skeleton"]
+  },
+  {
+    name:"video-modal-close",
+    files: ["lib/video-modal-close"]
   }
```

Comment:
This is loading three extra lines of code. Does it really make sense to make a separate request on the relevant pages vs just including it in the main bundle?

(FWIW, not something to fix at this time, as it's not causing an issue.)